module/help_mode.c

```diff
@@ -327,6 +327,11 @@ const char* help5[HELP5_LENGTH] = { "5/16 OPERATORS",
 									"     AND EACH SUBSEQUENT BIT",
 									"     IS +1 SEMITONE",
                                     " ",
+                                    "// N.B  D   GET DEGREE IN SCALE"
```

Comment:
don't need `//` before the ops, it's used to separate sections

Reply:
Ok, thanks